package.json

```diff
@@ -67,6 +67,8 @@
     "icons:build:optimize": "csso -i ./dist/clr-icons/clr-icons.css -o ./dist/clr-icons/clr-icons.min.css -s file;",
     "icons:build:package": "cpy ./npm/clr-icons/package.json ./npm/clr-icons/README.md ./dist/clr-icons/; replace '@VERSION' $npm_package_version ./dist/clr-icons/package.json",
     "icons:build:svg": "node ./scripts/clr-icons-svg.js",
+    "icons:test": "ng test clr-icons --watch=false",
```

Comment:
Can you also add this to the `test` script? 
`"test": "npm-run-all angular:test core:test icons:test",`